go.mod

```diff
@@ -6,8 +6,9 @@ require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.2.0
 	github.com/frankban/quicktest v1.11.3 // indirect
 	github.com/gardener/cert-management v0.6.0
-	github.com/gardener/gardener v1.32.0
+	github.com/gardener/gardener v1.33.1
 	github.com/go-logr/logr v0.4.0
+	github.com/gobuffalo/packr/v2 v2.5.1
```

Comment:
Why is packr added again?
It has been removed with #69 and should not be needed anymore.

Reply:
Thanks for noticing, not sure what happened here. I guess I had an old version with `packr` and even though I followed what I think is the correct way to revendor (`git pull`, `make revendor`, `make generate`, etc.), it somehow not only remained, but managed to pass all tests executed by `make verify`. Since I opened many revendor PRs at the same time I followed the same procedure for all of them but looked carefully at the changes for only some of them. Fixed now.